minerva-university/CourseSwaps
# configured testing with jest for react

Description:
I configured testing using Jest for JSX components and added tests for the API provider and UserContext files.


**Changes Made:**
- Change 1: I configured testing for react using Jest
- Change 2: I added tests for the API provider and UserContext files

**How to Test:**
- Step 1: run "npm install" to get the recent packages
- Step 2:  run "npm test" to run the testing suite
- Alternatively, install the jest extension on VSCode if you don't have it already. Check the testing tab and run all the tests. 


**Checklist:**
- I have performed a self-review of my code.
- I have commented my code, particularly in hard-to-understand areas.
- My changes generate no new warnings.
- I have added tests that prove my fix is effective or that my feature works.
- New and existing unit tests pass locally with my changes.
